### ISSUE TITLE
Loop not picked up by screen reader in start.ejs

### ISSUE THREAD

**Issue by koenblezer** [chelseadoeleman/route-planner#1]

**Comment by koenblezer** [comment]
When tabbing through website with a screen reader it navigates to the finish button and doesn't read the specific steps to the user. Perhaps change the type of element being used here.

**Comment by koenblezer** [comment]
`<li>
                    <a href="/start" class="current">Home</a>
                </li>
                <% json.steps.forEach(function (step, i) { %>
                    <li>
                        <a href="<%= nextRoute %>#<%= i + 1 %>" class="current"><%= i + 1 %></a>
                    </li>
                <% }) %>
                <li>
                    <a href="<%= finishRoute %>">Finish</a>
                </li>`
